scripts/splash/index.tsx

```diff
@@ -154,6 +154,14 @@ const App = () => {
   return (
     <React.Fragment>
       <Box marginBottom={1} flexDirection="column">
+        <Box>
+          <Box width={3}>💡</Box>
+          <Box>
+            Tip: disable <Text bold>yarn splash</Text> by running{' '}
+            <Text bold>yarn dev-no-splash</Text> or setting an environment
+            variable <Text bold>DISABLE_SPLASH=1</Text>
```

Comment:
Isn't it a bit strange to have this message here when the responsibility for this condition is handled by the caller? This means that running `yarn splash` will show this message.

Reply:
People need to learn about it somehow or we're always going to get messages from people like "hey how do I turn off yarn splash". Even if we let people on Slack know, there's outside contributors and new people joining all the time.

I see what you mean about this showing up when people run the standalone command. I can make it show up when storybook is running, but not when it's run by itself. How about that?

Reply:
I totally agree that showing a message is good for awareness 👍 

My question was around whether it's appropriate to have the message living in `scripts/splash/`, when the code that handles what's advertized by the message lives elsewhere (in the webpack config).

Alternatively, would it make more sense to handle the conditional for `DISABLE_SPLASH` in `scripts/splash/index.ts`, so both the message and the condition live under the same roof?

Reply:
I tried doing it in the webpack config but webpack overrides stdout so I can't just echo.

I could create a separate script for it but I think that would be adding too much complexity for very little gain